src/test/java/org/jboss/xavier/analytics/rules/workload/inventory/FlagsTest.java

```diff
@@ -374,6 +374,93 @@ public void test_CpuAffinityNotNull()
 
     }
 
+    public void test_VMDRS_VMHA_BALLOONEDMEM_ENCRYPTEDDISK_OPAQUENETWORK_true() {
+        Map<String, Object> facts = new HashMap<>();
+        // always add a String fact with the name of the agenda group defined in the DRL file (e.g. "SourceCosts")
+        facts.put("agendaGroup", "Flags");
+
+        VMWorkloadInventoryModel vmWorkloadInventoryModel = new VMWorkloadInventoryModel();
+        vmWorkloadInventoryModel.setHasVmDrsConfig(true);
+        vmWorkloadInventoryModel.setHasVmHaConfig(true);
+        vmWorkloadInventoryModel.setBalloonedMemory(1000);
+        vmWorkloadInventoryModel.setHasEncryptedDisk(true);
+        vmWorkloadInventoryModel.setHasOpaqueNetwork(true);
+        facts.put("vmWorkloadInventoryModel", vmWorkloadInventoryModel);
+
+        WorkloadInventoryReportModel workloadInventoryReportModel = new WorkloadInventoryReportModel();
+        facts.put("workloadInventoryReportModel",workloadInventoryReportModel);
+
+        Map<String, Object> results = createAndExecuteCommandsAndGetResults(facts);
+        Assert.assertEquals(2, results.get(NUMBER_OF_FIRED_RULE_KEY));
+        Utils.verifyRulesFiredNames(this.agendaEventListener, "AgendaFocusForTest", "Flag_Opaque_Network", "Flag_Encrypted_Disk", "Flag_Ballooned_Memory", "Flag_VM_HA_Config", "Flag_VM_DRS_Config");
```

Comment:
Has something changed here? It looks to me like 5 rules fire but the assertion of 2 is successful. I don't understand.

Reply:
You are totally right !!!! And the trick is the missing @Test annotation in the last 3 methods.
Good catch :) Thank you